### PR TITLE
[FLINK-11564] Translate "How To Contribute" page into Chinese

### PR DESCRIPTION
**Pull request by XuQianJin-Stars**
Translate "How To Contribute" page into Chinese.

The markdown file is located in: flink-web/how-to-contribute.zh.md
The url link is: https://flink.apache.org/zh/how-to-contribute.html

Please adjust the links in the page to Chinese pages when translating.

### CODE REVIEW

**Comment by kaibozhou** [comment]
Thanks for the effort.

Some formatting issues can refer to the Flink Translation Specifications:
https://docs.google.com/document/d/1zhGyPU4bVJ7mCIdUTsQr9lHLn4vltejhngG_SOV6g1w/edit#heading=h.by2zo7lwgzr5

**Comment by XuQianJin-Stars** [comment]
hi @kaibozhou Thank you very much. I will modify it according to the specification.
best 
qianjin

File: how-to-contribute.zh.md [hunk 1]
```diff
@@ -4,146 +4,141 @@ title: "如何参与贡献"
...
 -----
 
-## Contribute code
+## 贡献代码
 
-Apache Flink is maintained, improved, and extended by code contributions of volunteers. The Apache Flink community encourages anybody to contribute source code. In order to ensure a pleasant contribution experience for contributors and reviewers and to preserve the high quality of the code base, we follow a contribution process that is explained in our [Contribute code]( {{ site.base }}/contribute-code.html) guide. The guide also includes instructions on how to setup a development environment, our coding guidelines and code style, and explains how to submit a code contribution.
+Apache Flink 通过志愿者的代码贡献得到维护，改进和扩展。 Apache Flink 社区鼓励任何人贡献源代码。为了确保贡献者和评审者有一个愉快的贡献体验并保持高质量的代码库，我们遵循我们的[贡献代码]( {{ site.base }}/contribute-code.html)指南。 该指南还包括有关如何设置开发环境，编码指南和代码样式的说明，并说明如何提交代码贡献。
 
-**Please read the [Contribute code]( {{ site.base }}/contribute-code.html) guide before you start to work on a code contribution.**
+**在开始处理代码贡献之前，请阅读[贡献代码]( {{ site.base }}/contribute-code.html)指南。**
```
Comment (wuchong) [inline how-to-contribute.zh.md]: 处理代码贡献 -> 贡献代码

File: how-to-contribute.zh.md [hunk 2]
```diff
@@ -4,146 +4,141 @@ title: "如何参与贡献"
...
 
-Please do also read the [Submit a Contributor License Agreement]({{ site.baseurl }}/how-to-contribute.html#submit-a-contributor-license-agreement) Section.
+另请阅读[提交贡献者许可协议]({{ site.baseurl }}/how-to-contribute.html#submit-a-contributor-license-agreement)部分。
 
-### Looking for an issue to work on?
+### 如何寻找可以解决的问题？
```
Comment (wuchong) [inline how-to-contribute.zh.md]: 正在寻找一个可以参与工作的问题？

File: how-to-contribute.zh.md [hunk 3]
```diff
@@ -4,146 +4,141 @@ title: "如何参与贡献"
...
 {:.no_toc}
 
-We maintain a list of all known bugs, proposed improvements, and suggested features in [Flink's Jira](https://issues.apache.org/jira/browse/FLINK/?selectedTab=com.atlassian.jira.jira-projects-plugin:issues-panel). Issues that we believe are good tasks for new contributors are tagged with a special "starter" tag. Those tasks are supposed to be rather easy to solve and will help you to become familiar with the project and the contribution process.
+我们在 [Flink's Jira](https://issues.apache.org/jira/browse/FLINK/?selectedTab=com.atlassian.jira.jira-projects-plugin:issues-panel) 中保留了所有已知错误，建议的改进和建议功能的列表。 我们将对于新贡献者有帮助的问题标记为特殊的 “starter” 标记。 这些任务应该很容易解决，并将帮助您熟悉项目和贡献过程。
```
Comment (wuchong) [inline how-to-contribute.zh.md]: Flink's Jira -> Flink Jira
保留 -> 维护
bug 可以不翻译
我们将对于新贡献者有帮助的问题标记为特殊的 “starter” 标记  -> 对于新贡献者友好的问题，我们将其打上了特殊的 "starter" 标记。

File: how-to-contribute.zh.md [hunk 4]
```diff
@@ -4,146 +4,141 @@ title: "如何参与贡献"
...
 
-Please have a look at the list of [starter issues](https://issues.apache.org/jira/issues/?jql=project%20%3D%20FLINK%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20starter%20ORDER%20BY%20priority%20DESC), if you are looking for an issue to work on. You can of course also choose [any other issue](https://issues.apache.org/jira/issues/?jql=project%20%3D%20FLINK%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20priority%20DESC) to work on. Feel free to ask questions about issues that you would be interested in working on.
+请查看[入门问题](https://issues.apache.org/jira/issues/?jql=project%20%3D%20FLINK%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20starter%20ORDER%20BY%20priority%20DESC)列表，如果您正在寻找可以解决的问题。 您当然也可以选择[任何其他问题](https://issues.apache.org/jira/issues/?jql=project%20%3D%20FLINK%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20priority%20DESC)继续工作。 您可以随意提出有关您有兴趣处理的问题。
```
Comment (wuchong) [inline how-to-contribute.zh.md]: 入门问题 -> starter issue    (此处可以不翻译，可以和上文对应)

File: how-to-contribute.zh.md [hunk 5]
```diff
@@ -4,146 +4,141 @@ title: "如何参与贡献"
...
 
-## Contribute documentation
+## 贡献文档
 
-Good documentation is crucial for any kind of software. This is especially true for sophisticated software systems such as distributed data processing engines like Apache Flink. The Apache Flink community aims to provide concise, precise, and complete documentation and welcomes any contribution to improve Apache Flink's documentation.
+良好的文档对任何类型的软件都至关重要。 对于复杂的软件系统尤其如此，例如 Apache Flink 等分布式数据处理引擎。 Apache Flink社区旨在提供简明，精确和完整的文档，并欢迎任何改进 Apache Flink 文档的贡献。
 
-- Please report missing, incorrect, or outdated documentation as a [Jira issue](http://issues.apache.org/jira/browse/FLINK).
-- Flink's documentation is written in Markdown and located in the `docs` folder in [Flink's source code repository]({{ site.baseurl }}/community.html#main-source-repositories). See the [Contribute documentation]({{ site.base }}/contribute-documentation.html) guidelines for detailed instructions for how to update and improve the documentation and to contribute your changes.
+- 请将缺失，不正确或过时的文档报告为一个 [Jira 问题](http://issues.apache.org/jira/browse/FLINK)。
```
Comment (wuchong) [inline how-to-contribute.zh.md]: 如果发现文档缺失、不正确或过时的问题，可以提交一个 [Jira issue]

File: how-to-contribute.zh.md [hunk 6]
```diff
@@ -4,146 +4,141 @@ title: "如何参与贡献"
...
+- Flink 的文档是用 Markdown 编写的，位于[ Flink 的源代码库]({{ site.baseurl }}/community.html#main-source-repositories)的 `docs` 文件夹中。 有关如何更新和改进文档以及提供更改的详细说明，请参阅[贡献文档]({{ site.base }}/contribute-documentation.html) 指南。
```
Comment (wuchong) [inline how-to-contribute.zh.md]: Flink 的  -> "的"可省略，语句会简明些。

File: how-to-contribute.zh.md [hunk 7]
```diff
@@ -4,146 +4,141 @@ title: "如何参与贡献"
...
+- 向访问者介绍 Apache Flink 及其功能。
+- 鼓励访问者下载和使用 Flink 。
+- 鼓励访客与社区互动。
 
-We welcome any contribution to improve our website.
+我们欢迎任何改进我们网站的贡献。
 
-- Please open a [Jira issue](http://issues.apache.org/jira/browse/FLINK) if you think our website could be improved.
-- Please follow the [Improve the website]({{ site.baseurl }}/improve-website.html) guidelines if you would like to update and improve the website.
+- 如果您认为我们的网站可以改进，请打开一个 [Jira issue](http://issues es.apache.org/jira/browse/flink) 。
```
Comment (wuchong) [inline how-to-contribute.zh.md]: 打开 -> "提交" or "创建"

File: how-to-contribute.zh.md [hunk 8]
```diff
@@ -4,146 +4,141 @@ title: "如何参与贡献"
...
-> For a corporation that has assigned employees to work on an Apache project, a [Corporate CLA](http://www.apache.org/licenses/cla-corporate.txt) (CCLA) is available for contributing intellectual property via the corporation, that may have been assigned as part of an employment agreement. Note that a Corporate CLA does not remove the need for every developer to sign their own CLA as an individual, to cover any of their contributions which are not owned by the corporation signing the CCLA.
+> 对于分配员工参与 Apache 项目的公司，可以通过公司提供[Corporate CLA](http://www.apache.org/licenses/cla-corporate.txt) (CCLA)来贡献知识产权，该知识产权可能是作为雇佣协议的一部分分配的。请注意，公司类并不免除每个开发人员作为个人签署他们自己的类的需要，以覆盖他们的贡献，而这些贡献并不属于签署 CCLA 的公司。
 >
->  ...
+> ...
 
 -----
 
-## How to become a committer
+## 如何成为一个提交者
```
Comment (wuchong) [inline how-to-contribute.zh.md]: 提交者 -> 提交者（Committer）

File: how-to-contribute.zh.md [hunk 9]
```diff
@@ -4,146 +4,141 @@ title: "如何参与贡献"
...
 
-Committers are community members that have write access to the project's repositories, i.e., they can modify the code, documentation, and website by themselves and also accept other contributions.
+提交者是具有对项目存储库的写访问权的社区成员，即他们可以自己修改代码，文档和网站，也接受其他贡献。
```
Comment (wuchong) [inline how-to-contribute.zh.md]: 存储库  -> 仓库
写访问权 -> 写权限

**Comment by XuQianJin-Stars** [comment]
hi @wuchong  Thank you very much for your review. I have repaired a version according to the review.
best
qianjin

**Comment by XuQianJin-Stars** [comment]
hi @wuchong @kaibozhou Thank you very much, all comments has addressed.
best
qianjin

**Pull request closed without merging**